qa/L0_input_validation/models/input_all_optional/config.pbtxt

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Please always use "NVIDIA CORPORATION & AFFILIATES" for any new files or when updating years on existing files, i.e. "Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved."

We used to just say "NVIDIA CORPORATION", but there was an update years back that switched over to "NVIDIA CORPORATION & AFFILIATES".